onnx/version_converter/helper.cc

```diff
@@ -16,7 +16,7 @@ namespace ONNX_NAMESPACE { namespace version_conversion {
             input2_sizes[i].dim != 1) return -1;
         if (input2_sizes[i].dim != input1_sizes[axis + i].dim) broadcast = true;
       }
-      // Return true if broadcasting is required
+      // Return 1 if broadcasting is required
```

Comment:
curious why check_numpy_unibroadcastable_and_require_broadcast does not return `bool`